sdk/core/azure-core-amqp/test/ut/CMakeLists.txt

```diff
@@ -31,6 +31,22 @@ add_executable(azure-core-amqp-tests
 )
 add_dependencies(azure-core-amqp-tests azure-core-amqp)
 
+if (MSVC)
+  # Disable warnings:
+  # - C26495: Variable
+  #             - 'testing::internal::Mutex::critical_section_'
+  #             - 'testing::internal::Mutex::critical_section_init_phase_'
+  #             - 'testing::internal::Mutex::owner_thread_id_'
+  #             - 'testing::internal::Mutex::type_'
+  #           is uninitialized. Always initialize member variables (type.6).
+  # - C26812: The enum type
+  #             - 'testing::internal::Mutex::StaticConstructorSelector'
+  #             - 'testing::TestPartResult::Type'
+  #           is unscoped. Prefer 'enum class' over 'enum' (Enum.3)
+  # - C6326: Google comparisons
+  target_compile_options(azure-core-amqp-tests PUBLIC /wd26495 /wd26812 /wd6326 /wd28204 /wd28020 /wd6330 /wd4389)
```

Comment:
These are similar to https://github.com/Azure/azure-sdk-for-cpp/blob/5a348b6766cc58903a3c8ee2effc841ae06194c9/sdk/core/azure-core/test/ut/CMakeLists.txt#L107

Reply:
It should be identical (they are cut and pasted).

What is different?

Reply:
It **is** identical.